src/js/form-builder.js

```diff
@@ -1012,6 +1059,62 @@ const FormBuilder = function (opts, element, $) {
     }
   }
 
+  function setupSortableWrapper(rowWrapperNode) {
+    $(rowWrapperNode).sortable({
+      connectWith: '.rowWrapper',
+      cursor: 'move',
+      opacity: 0.9,
+      revert: 150,
+      cursorAt: {
+        left: 5,
+        top: 5,
+      },
+      placeholder: 'ui-state-highlight',
+      grid: [1, 1],
+      stop: function (event, ui) {
+        autoSizeRowColumns(ui.item.closest('.rowWrapper'))
+      },
+      update: function (event, ui) {
+        syncFieldWithNewRow(ui.item.attr('id'))
+      },
+    })
+  }
+
+  function prepareFieldRow(data) {
+    let result = {}
+
+    result = h.tryParseColumnInfo(data)
+    TryCreateNew()
+
+    if (!formRows.includes(result.rowNumber)) {
+      formRows.push(result.rowNumber)
+    }
+
+    return result
+
+    function TryCreateNew() {
```

Comment:
This could be moved out of `prepareFieldRow` nesting and take result as an argument

Reply:
It could but I didn't really want to make it look like it was a function that was usable outside of prepareFieldRow. The only reason I nested it was to just make the overall function cleaner. This is something I am used to in c# but it doesn't bother me to pull it out if that's the style of the codebase